java/RSPEC-1317.java

```diff
@@ -0,0 +1,6 @@
+class StringBuilderChat {
```

Comment:
RSPEC-2176: Class has same name as parent
<!-- lookout footnote separator -->
_Comment made by the analyzer sonarcheck_analyzer._